src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7240/poll.immediately.with.wrong.url.if.unsafe.request.invalidates.cache/accept/client.rpt

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property response1Date ${http_cache:date()} # sharedProperty
+property response2Date ${http_cache:datePlus(5)} # sharedProperty
+property etag ${http_cache:strongEtag()} #shared
+property etag2 ${http_cache:strongEtag()} #shared
+
+connect await ROUTED_PROXY
+        "nukleus://streams/http-cache#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/items/1")
+                              .header(":authority", "localhost:8080")
+                              .header("prefer", "wait=50")
+                              .header("cache-control", "max-age=0")
+                              .header("if-none-match", etag)
+                              .build()}
+connected
+
+write close
+
+
+connect await SECOND_RESPONSE_SENT
+        "nukleus://streams/http-cache#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "POST")
+                              .header(":path", "/items/1")
+                              .header(":authority", "localhost:8080")
+                              .header("content-length", "9")
+                              .build()}
+connected
+
+write "post body"
+
+write close
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "200")
+                             .header("content-length", "11")
+                             .header("date", response2Date)
+                             .header("link", "<http://wronghost.org:8080/items>; rel=\"collection\"")
```

Comment:
This scenario needs to demonstrate that the `link` response header is ignored because it is cross-origin.
To prove it is being ignored for the right reasons, we need to have an existing poll loop in flight for the "wronghost.org" url, but _not_ trigger the cache invalidation or immediate poll when this `link` response header is received.

Therefore, we may want to use other hostnames, such as `example.com` and `example.net`, per RFC 2606.